metabrainz/templates/users/mb-signup.html

```diff
@@ -3,17 +3,125 @@
-       class="btn btn-primary btn-lg btn-block">{{ _('Sign up with MusicBrainz') }}</a>
+            <div class="form-group">
+              <label class="form-label" htmlFor="yourPassword">
+                Password
+              </label>
+              <input
+                class="form-control"
+                id="yourPassword"
+                name="password"
+                required
+                type="password"
+              />
+            </div>
+            <div class="form-group">
+              <label class="form-label" htmlFor="confirmPassword">
+                Confirm password
+              </label>
+              <input
+                class="form-control"
+                id="confirmPassword"
+                name="confirm-password"
+                required
+                type="password"
+              />
+            </div>
+
+            <div>
+                Please note that your contributions will be released into the public domain or licensed for use.
+                We will never share your personal information.
+              <br/>
+              <a class="small" href="#" data-toggle="modal" data-target="#conditions-modal" data-backdrop="false">
+                Click here to read more
+              </a>
+            </div>
+            
+            <button class="btn btn-primary main-action-button" type="submit">
+              Create account
+            </button>
+          </form>
+          <div class="modal fade" id="conditions-modal">
+            <div class="modal-dialog" role="document">
+              <div class="modal-content">
+                <p>Note that any contributions you make to MusicBrainz will be released into the Public Domain and/or licensed under a Creative Commons by-nc-sa license.
+                  Furthermore, you give the MetaBrainz Foundation the right to license this data for commercial use.
+                  <br/>Please read our <a>license page</a> for more details.
+                <p>MusicBrainz believes strongly in the privacy of its users. Any personal information you choose to provide will not be sold or shared with anyone else.
+                  <br/>Please read our <a>privacy policy</a> for more details.</p>
+                <p>You may remove your personal information from our services anytime by deleting your account.
+                  <br/>Please read our <a>GDPR compliance statement</a> for more details.</p>
+                <p>Creating an account on MetaBrainz will give you access to accounts on our other services, such as ListenBrainz, MusicBrainz, BookBrainz, and more.
+                  <br/>We do not automatically create accounts for these services when you create a MetaBrainz account, but you will only be a few clicks away doing so.</p>
+                  <button class="btn btn-primary center-block" type="button" data-dismiss="modal">
+                    Sounds good
+                  </button>
```

Comment:
Why have a separate button for this? If we do, shouldn't we have a second button for "I don't like this, get me out of here"? :D But honestly just "hide" seems better to me than "sounds good", in theory.